sql/moz-fx-data-shared-prod/org_mozilla_reference_browser_derived/baseline_clients_last_seen_v1/checks.sql

```diff
@@ -0,0 +1,35 @@
+-- Overrides bigquery_etl.glean_usage due to this app having very limited data, sometimes none
+
+#warn
+{{ is_unique(["client_id"], where="submission_date = @submission_date") }}
+
+#warn
+{{ not_null([
+  "submission_date",
+  "client_id",
+  "sample_id",
+  "first_seen_date",
+  "days_seen_bits",
+  "days_active_bits",
+  "days_created_profile_bits",
+  "days_seen_session_start_bits",
+  "days_seen_session_end_bits"
+  ], where="submission_date = @submission_date") }}
+
+#warn
+SELECT
+  IF(
+    COUNTIF(normalized_channel NOT IN ("nightly", "aurora", "release", "Other", "beta", "esr")) > 0,
+    ERROR("Unexpected values for field normalized_channel detected."),
+    NULL
+  )
+FROM
+  `{{ project_id }}.{{ dataset_id }}.{{ table_name }}`
+WHERE
+  submission_date = @submission_date;
+
+#warn
+{{ matches_pattern(column="country", pattern="^[A-Z]{2}$", where="submission_date = @submission_date") }}
```

Comment:
I was looking at the failed check and I think this check is the one failing because of division by zero (job id `bqjob_r1ec1c7ce8bcc79_000001904d5a57c0_1`) so I think this needs to be taken out as well